thirdparty/libktx/lib/texture2.c

```diff
@@ -2107,21 +2239,34 @@ ktxTexture2_IterateLoadLevelFaces(ktxTexture2* This, PFNKTXITERCB iterCb,
                 ZSTD_ErrorCode error = ZSTD_getErrorCode(levelSize);
                 switch(error) {
                   case ZSTD_error_dstSize_tooSmall:
-                    return KTX_INVALID_VALUE; // inflatedDataCapacity too small.
+                    return KTX_DECOMPRESS_LENGTH_ERROR; // inflatedDataCapacity too small.
+                  case ZSTD_error_checksum_wrong:
+                    return KTX_DECOMPRESS_CHECKSUM_ERROR;
                   case ZSTD_error_memory_allocation:
                     return KTX_OUT_OF_MEMORY;
                   default:
                     return KTX_FILE_DATA_ERROR;
                 }
             }
+
             // We don't fix up the texture's dataSize, levelIndex or
             // _requiredAlignment because after this function completes there
             // is no way to get at the texture's data.
             //nindex[level].byteOffset = levelOffset;
             //nindex[level].uncompressedByteLength = nindex[level].byteLength =
                                                                 //levelByteLength;
+        } else if (This->supercompressionScheme == KTX_SS_ZLIB) {
+            result = ktxUncompressZLIBInt(uncompressedDataBuf,
```

Comment:
Yet another solution would be to write a Godot-specific file with definitions for functions `ktxCompressZLIBBounds`, `ktxCompressZLIBInt` and `ktxUncompressZLIBInt` so that we don't need to include `basisu_miniz.h`, but that seems like an unnecessary effort and there should be a better way to solve this.